src/precice/impl/Participant.cpp

```diff
@@ -6,6 +6,7 @@
 #include "DataContext.hpp"
 #include "MappingContext.hpp"
 #include "MeshContext.hpp"
+#include "ReadDataContext.hpp"
```

Comment:
Looks odd. Why is only one of both headers needed here?

Reply:
Removed in ce0eb84.